### PR TITLE
90728208: Remove rejection from the UI

### PR DESCRIPTION
**Pull request by cmc333333**
* Adds a prominent link on the email to comment; removes reject UI (though backend still exists)
* Said link jumps to the comments anchor
* Uses consistent "Comment" language

screen shot 2015-03-23 at 12 55 40 pm
screen shot 2015-03-23 at 1 03 32 pm

### CODE REVIEW

File: app/views/carts/_approval_actions.html.erb [hunk 1]
```diff
@@ -4,10 +4,5 @@
         approval_response_path(approver_action: 'approve',
                                cart_id: cart.id,
                                ), class: 'form-button' %>
-
-    <%= link_to "Reject",
-        approval_response_path(approver_action: 'reject',
-                               cart_id: cart.id,
-                               ), class: 'reject-link' %>
```
Comment (afeld) [inline app/views/carts/_approval_actions.html.erb]: Are we ok with removing this for the non-NCR use cases?

**Comment by afeld** [comment]
Just the one overall question (probably for @phirefly)...otherwise :ship:

**Comment by phirefly** [comment]
Let's rename `Comment` to `Send a Comment`
Otherwise, Ship it!

File: app/views/carts/_approval_actions.html.erb (hunk 1 above)
Comment (phirefly) [inline app/views/carts/_approval_actions.html.erb]: Navigator folks have not been engaged, so let's go ahead and remove altogether.

**Comment by cmc333333** [comment]
Updated language + downgraded commenting from email to a link
screen shot 2015-03-24 at 2 17 18 pm
screen shot 2015-03-24 at 2 16 52 pm

**Pull request merged**